qr-code-auth/README.md

```diff
@@ -0,0 +1,52 @@
+# QrCodeAuth
+
+## Requests
+
+1. POST 
+
+`/qr-code-auth?user-id=${user-id}&device-info={deviceInfo}`
```

Comment:
Nit: A better URL path I feel would be`/qr-code-authentication`. It is more descriptive and accurate than `/qr-code-auth`.